src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -193,110 +246,93 @@ export function useSwapInputsController({
         }
       }
 
-      isFetching.value = false;
       quote.value = data;
-      runOnJS(updateQuoteStore)(data);
 
       if (!data || (data as QuoteError)?.error) {
         return;
```

Comment:
- when there is no data or if there is an error, we are returning here without updating the isFetching.value
- as for runOnJS(updateQuoteStore): who needs the quote store? and if there was an error, do we want to update the store accordingly or no?

Reply:
Yep this is exactly why I was always setting the isFetching value here.